scripts/index.js

```diff
@@ -0,0 +1,133 @@
+const state = {
+    tempCount: 70
+};
+
+const increaseTemp = () => {
+    state.tempCount += 1
+    const count = document.querySelector("#temperature");
+    count.textContent = state.tempCount;
+
+    if (state.tempCount > 79) {
+        red();
+        hot();
```

Comment:
Each of these conditions is checking whether the set temperature is greater than some lower bound, and in the branch body each of the function pairs is setting the className of one element, and the contents of another.

What if we had an array that stored records made of that lower bound, the name of the class for the temperature, and the string to use to update the garden floor? Then after updating the temperature, we could iterate over our data structure, find the values we need, and up date the UI, all in a data driven fashion. Maybe the data could look like:

```js
const temperatureSettings = [
    { lowerBound: 79, color: 'red', ground: '🧗🏾____🌵🌵🌵__🐍__🌵🌵'},
    { lowerBound: 69, color: 'orange', ground: '💃🏿💃🏾💃🏽💃🏼💃🏻__🚴🏿_🚴🏾🚴🏽🚴🏼🚴🏻'},
    { lowerBound: 59, color: 'yellow', ground: '🌻🌻🌻__🌻🌻_🐿_🌷__🌷🌷'},
    { lowerBound: 49, color: 'green', ground: '🧗🏾____🌵🌻_🐿_🌷__🌷🌷'},
    { lowerBound: null, color: 'blue', ground: '🌲______⛄️🌲🌲🏂🏻🌲__🌲'},
];
```

How could we use this to simplify the update logic?